notebooks/external_convergence_shear_distributions_tutorial.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_As_from_sigma8(target_sigma8, Om0):
</pre><p>can you define what these definitions do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/sim-pipeline/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>